### PR TITLE
Release staging v.1.8.2

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,17 +3,17 @@ module github.com/mongodb/terraform-provider-mongodbatlas
...
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
+	github.com/hashicorp/hcl/v2 v2.16.2
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.26.1
 	github.com/mongodb-forks/digest v1.0.4
 	github.com/mwielbut/pointy v1.1.0
 	github.com/spf13/cast v1.5.0
-	github.com/zclconf/go-cty v1.12.1
-	go.mongodb.org/atlas v0.24.0
+	github.com/zclconf/go-cty v1.13.1
+	go.mongodb.org/atlas v0.23.1
```
Comment (andreaangiolillo) [inline go.mod]:
```suggestion
	go.mongodb.org/atlas v0.24.0
```

Comment (Zuhairahmed) [reply go.mod]: i think PR is not grabbing all changes made, i.e. i think this one is missing too - https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1104/files

Comment (martinstibbe) [reply go.mod]: synced up to correct SDK version

Comment (Zuhairahmed) [reply go.mod]: @andreaangiolillo just confirming that this PR now captures latest go-client version

**Review by Zuhairahmed — APPROVED** [review]
LGTM - @martinstibbe can you create auto-gen CHANGELOG? i will then create commit to update that one directly to master

**Pull request merged**